### ISSUE TITLE
1.18.2 need to disable some dungeon rooms as our server has some anti greefing stuff in place

### ISSUE THREAD

**Issue by TheIronwitch** [AllenSeitz/DimDungeons#57]
so needed a list of the rooms that use fire tick and explosions so I can turn them off with the config

**Comment by AllenSeitz** [comment]
Good idea. I'll get back to you on that. I know it has already been 3 weeks, I'm sorry.

So few rooms use fire tick that I might just rework those rooms.
